Taskfile.yaml

```diff
@@ -4,6 +4,10 @@ version: '3'
 env:
   CGO_ENABLED: 0
 
+vars:
+  DOCKER_PLATFORM:
+    sh: '[ `arch` == "arm64" ] && echo "linux/x86_64" || echo ""'
```

Comment:
based on the [docs](https://taskfile.dev/usage/#gos-template-engine), shouldn't this be `{{ARCH}}` instead ?

Reply:
fixed